### DISCUSSION TITLE
Analog counters between 9 and 0

### DISCUSSION THREAD

**Discussion by JensHoRi** [jomjol/AI-on-the-edge-device#894] [Q&A]
I noticed, that I often got errors with negative rates when analog counter is between 9 and 0. here an example:
![D5D0AE17-08D8-4B2E-8D20-DE441AAC37F1](https://user-images.githubusercontent.com/47488279/182495262-e4893fc0-16f1-474d-be3d-4fcc5ebb3936.png)

the about 9.6 is recognized as 9.1

and in Kombination with second analog is 6, ai reporting as raw value 1039.86xx

i often notice this if numbers are between 9 and 0.

anyone else saw same behavior? Is it problem with AI should have better training data? Or a bug in correction only between 9 and 0??

**Comment by jomjol** [comment]
This is within the error level of the trained network - not much to do about.

**Reply by JensHoRi** [reply]
Not believe it’s a training issue.
With 9.1 and 6.7 this should lead to a 0.96

but result is 8.6. 

As I noticed: reading 
9 / 1 is shown as 0.91
9 / 2 is shown as 0.92
9 / 3 is shown as 0.93
9 / 4 is shown as 0.94
9 / 5 is shown as 0.95
9 / 6 is shown as 0.86
9 / 7 is shown as 0.87
9 / 8 is shown as 0.88
9 / 9 is shown as 0.89

so if following value higher 5 the one before is one to small if it should be 9

**Reply by jomjol** [reply]
The number, that is shown in the above html page is not the pure detection number for the single number, so the single image is really identified as 9.1 instead of 9.6, which it should be. But maybe there is a bug in the combination algo as well.

**Reply by JensHoRi** [reply]
Well maybe my example is to short. 

9/1 means the first is 9.0 to 9.9 and the second counter is 0.0 to 0.9

so if the first is clearly read as 9 it becomes 8 if the next counter is higher then 5

it’s only between 9 and 0 I saw this behavior
